packages/app/src/stores/page.tsx

```diff
@@ -25,7 +26,7 @@ import {
 const { isPermalink: _isPermalink } = pagePathUtils;
 
 
-export const useSWRxCurrentPage = (initialData?: IPagePopulatedToShowRevision|null): SWRResponse<IPagePopulatedToShowRevision|null> => {
+export const useSWRxCurrentPage = (initialData?: IPagePopulatedToShowRevision | null): SWRResponse<IPagePopulatedToShowRevision | null> => {
```

Comment:
Thanks for this refactor, but we are experimenting with swr2 and I'm not sure who the most recent implementer is, so can you please move commit back?